File: engine/engine-model/src/main/scala/tools/packet/field/effect/DojoAnimation.scala
package tools.packet.field.effect

import net.opcodes.SendOpcode
import tools.packet.PacketInput

class DojoAnimation(private var _firstByte: Byte, private var _animation: String) extends PacketInput {
  def firstByte: Byte = _firstByte

  def animation: String = _animation

  override def opcode(): SendOpcode = SendOpcode.FIELD_EFFECT
}

File: engine/engine-model/src/main/java/tools/packet/field/effect/DojoAnimation.java
package tools.packet.field.effect;

import net.opcodes.SendOpcode;
import tools.packet.PacketInput;

public record DojoAnimation(Byte firstByte, String animation) implements PacketInput {
   @Override
   public SendOpcode opcode() {
      return SendOpcode.FIELD_EFFECT;
   }
}